frontend/src/Explorer/View/Address.purs

```diff
@@ -90,18 +90,27 @@ addressDetailRowItems (CAddressSummary address) lang =
       , label: translate (I18nL.common <<< I18nL.cAddress) lang
       , value: address ^. (caAddress <<< _CAddress)
       , mCurrency: Nothing
-    }
+      }
     , { id: "1"
       , label: translate (I18nL.common <<< I18nL.cTransactions) lang
       , value: show $ address ^. caTxNum
       , mCurrency: Nothing
-    }
+      }
     , { id: "2"
```

Comment:
`Id` as numbers? It's not a big deal, but maybe we can use something more readable?